### PR TITLE
Add testing to sign up and login form

### PR DESCRIPTION
**Pull request by 9gustin**
## Summary

Add testing to sign up form.
Add testing to login form

## Screenshots

-

## Trello Card

[Tests de login](https://trello.com/c/5niBr3DE/114-tests-de-login)

### CODE REVIEW

File: react/w-books/src/app/screens/Login/components/LoginForm/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import "@testing-library/jest-dom";
+import '@testing-library/jest-dom/extend-expect';
+import { MemoryRouter } from 'react-router-dom';
+import 'mutationobserver-shim';
+import { act } from 'react-dom/test-utils';
+
+import LoginForm from './';
+
+const CorrectUserData = {
+  email: 'agustin.vazquez@wolox.com.ar',
```
Comment (LucasZibell) [inline react/w-books/src/app/screens/Login/components/LoginForm/index.test.tsx:12]: Don't use personal information

File: react/w-books/src/app/screens/Login/components/LoginForm/index.test.tsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  password: '123456'
+};
+
+const InvalidCase1UserData = {
+  email: '',
+  password: ''
+};
+
+const InvalidCase2UserData = {
+  email: '********$%a#@wolox.com.ar',
```
Comment (LucasZibell) [inline react/w-books/src/app/screens/Login/components/LoginForm/index.test.tsx:22]: same, use mocked data

File: react/w-books/src/app/screens/SignUp/components/SignUpForm/index.test.tsx [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    fireEvent.change(inputConfirmPassword, { target: { value: user.password_confirmation } });
+    fireEvent.change(inputName, { target: { value: user.first_name } });
+    fireEvent.change(inputLastName, { target: { value: user.last_name } });
+  })
+
+  return {
+    mockSubmit,
+    formElement
+  }
+}
```
Comment (LucasZibell) [inline react/w-books/src/app/screens/SignUp/components/SignUpForm/index.test.tsx:148]: EOF

File: react/w-books/src/app/screens/SignUp/components/SignUpForm/index.test.tsx [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+    fireEvent.change(inputPassword, { target: { value: user.password } });
+    fireEvent.change(inputConfirmPassword, { target: { value: user.password_confirmation } });
+    fireEvent.change(inputName, { target: { value: user.first_name } });
+    fireEvent.change(inputLastName, { target: { value: user.last_name } });
+  })
+
+  return {
+    mockSubmit,
+    formElement
+  }
```
Comment (LucasZibell) [inline react/w-books/src/app/screens/SignUp/components/SignUpForm/index.test.tsx:147]: Don't use personal information on the test

File: react/w-books/src/app/screens/Login/components/LoginForm/index.test.tsx [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  await act(async ()=> {
+    fireEvent.change(inputEmail, { target: { value: user.email } });
+    fireEvent.change(inputPassword, { target: { value: user.password } });  
+  })
+
+  return {
+    mockLogin,
+    loginForm
+  }
+}
```
Comment (LucasZibell) [inline react/w-books/src/app/screens/Login/components/LoginForm/index.test.tsx:95]: EOF

**Review by LucasZibell — CHANGES_REQUESTED** [review]

**Comment by 9gustin** [comment]
changes are ready for review

**Review by DavidRojas1612 — APPROVED** [review]

**Review by LucasZibell — APPROVED** [review]

**Pull request merged**